trpcultivate_phenotypes/src/Plugin/Validators/ValidHeaders.php

```diff
@@ -24,19 +19,13 @@
  */
 class ValidHeaders extends TripalCultivatePhenotypesValidatorBase implements ContainerFactoryPluginInterface {
 
-  /**
-   * This validator requires the following validator traits:
-   * - Headers - getHeaders: get all headers.
-   * - ColumnCount - getExpectedColumns: get the expected number of columns and strict comparison flag.
-   */
-  use Headers, ColumnCount;
-
-  /**
-   * Constructor.
+  /* Validator Traits required by this validator.
```

Comment:
Docblock should start with a opening line.
```suggestion
  /**
   * Validator Traits required by this validator.
```

Reply:
Addressed in commit [9c6f567](https://github.com/TripalCultivate/TripalCultivate-Phenotypes/pull/119/commits/9c6f567d684d387d84a9610e068af53466e3d95f) 👍